### PR TITLE
Use the new is_monitored "nobuild" flag.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: hotness/consumers.py [hunk 1]
```diff
@@ -213,6 +214,11 @@ def _handle_anitya_update(self, upstream, package, msg):
             self.publish("update.bug.file", msg=dict(
                 trigger=msg, bug=dict(bug_id=bz.bug_id)))
 
+            if is_monitored == 'nobuild':
+                self.log.info("Monitor flag set to 'nobuild'.  "
+                              "Skipping scratch build.")
```
Comment (pypingou) [inline hotness/consumers.py]: Do we want to ``self.publish("update.drop", msg=dict(trigger=msg, reason="pkgdb"))`` here as well?

**Comment by pypingou** [comment]
One question, otherwise :+1:

File: hotness/consumers.py (hunk 1 above)
Comment (ralphbean) [inline hotness/consumers.py]: Nah.  They have to turn this on explicitly -- so they should know already.

**Comment by ralphbean** [comment]
Thanks @pypingou!

**Pull request merged**